appveyor.yml

```diff
@@ -130,9 +131,15 @@ test_script:
     } else {
       $env:psqlopt="--bindir"
     }
+    if ("$env:pg" -eq "master") {
```

Comment:
What happens when 12 is no longer new ?

Reply:
I highly suggest we merge tests back. We can use inline `DO` with plpgsql and/or plr to work around version specific issues. [There](https://codecov.io/gh/mlt/plr/src/inline/pg_rsupport.c#L777) are [few](https://codecov.io/gh/mlt/plr/src/inline/pg_userfuncs.c#L514) tests for existing functionality that are [missing](https://codecov.io/gh/mlt/plr/src/inline/pg_conversion.c#L1262) and I don't feel like either duplicating those or starting a new test file.